src/CoinEntry.h

```diff
@@ -27,37 +27,38 @@ typedef std::vector<std::pair<Data, Data>> HashPubkeyList;
 /// Implement this for all coins.
 class CoinEntry {
```

Comment:
since we have plan to refactor `CoinEntry`, let's add `#pragma clang diagnostic` to ignore those `unused` warnings

Reply:
we can also silent for this file using CMake I believe